Diff/Diff.sublime-syntax

```diff
@@ -25,12 +25,13 @@ contexts:
         1: punctuation.definition.separator.diff
     - match: ^\d+(,\d+)*(a|d|c)\d+(,\d+)*$\n?
       scope: meta.diff.range.normal
-    - match: ^(@@)\s*(.+?)\s*(@@)($\n?)?
+    - match: ^(@@)\s*(.+?)\s*(@@)\s*(.*)\s*$\n?
```

Comment:
```suggestion
    - match: ^(@@)\s*(.+?)\s*(@@)\s*(.*?)\s*$\n?
```

The group match must be non-greedy, otherwise the following `\s*` will never match and everything is consumed in the group already.

Reply:
Thanks.